core/src/main/java/com/graphhopper/routing/util/PriorityCode.java

```diff
@@ -42,11 +42,11 @@ public int getValue() {
         return value;
     }
 
-    /**
-     * This method returns the PriorityCode.value in a range between 0 and 1 suitable for direct usage in a Weighting.
-     */
-    public static double getFactor(int val) {
-        return (double) val / BEST.getValue();
+    public static double getFactor(int value) {
+        return (double) value / 10.0;
```

Comment:
Maybe put `value / UNCHANGED` here ?

Reply:
IMO this is unrelated to the PriorityCode itself and only how we store them.